src/styles/blocks/catalog.scss

```
@@ -0,0 +1,23 @@
+.catalog{
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px $catalog-padding;
+  margin: auto;
+
+  width: $catalog-width;
+  &__card {
+    margin-right: $card-margin-right;
+  }
+  &__card:nth-child(4n) {
+    margin-right: 0;
+  }
+  &__card:nth-child(-n + 4) {
+    margin-bottom: 46px;
+  }
```

You can move this selector to &__card selector

```suggestion

  &:nth-child(4n) {
    margin-right: 0;
  }
  &:nth-child(-n + 4) {
    margin-bottom: 46px;
  }

```

---

Fix